petrinet/examples/sir.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "SIR Model",
-  "schema": "https://raw.githubusercontent.com/DARPA-ASKEM/Model-Representations/petrinet_v0.1/petrinet/petrinet_schema.json",
+  "schema": {
+    "schema_uri": "https://raw.githubusercontent.com/DARPA-ASKEM/Model-Representations/petrinet_v0.1/petrinet/petrinet_schema.json",
+    "framework": "Petri Net",
```

Comment:
Would it make sense to make the `framework` equal to the `ID` (`petrinet`) rather than the spelled out name (`Petri Net`) of the framework?

Reply:
Yeah I think that's a good point. We may want in the HMI some human readable name for these frameworks and right now those human readable names don't exist anywhere. That could be another key?

Reply:
Yes, a human readable name for each framework would be useful but I feel like that's model-independent so it would be better to define it as part of the framework rather than each individual model.

Reply:
I agree with a human readable name of the framework being stored upstream, but should the name of the framework and the version be as well? Doesn't this give more fields that can get out of sync? It seems like given the schema url there is a unique value for both `framework` and `version` which makes me think it shouldn't be here.